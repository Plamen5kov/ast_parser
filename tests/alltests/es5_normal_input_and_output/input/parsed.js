(function () {
    var MyButton = android.widget.Button2222.extend("my.custom.ClassFromOriginalTest", {
        onClick222222: function () {
            console.log('click happened');
        },
        onClick222222222222: function () {
            console.log('click happened');
        }
    });
})();