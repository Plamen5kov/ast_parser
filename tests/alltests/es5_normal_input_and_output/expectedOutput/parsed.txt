Java File: my.custom.ClassFromOriginalTest - Extend Class: android.widget.Button2222 - Overridden Methods: onClick222222,onClick222222222222